contracts/src/PBHEntryPointImplV1.sol

```diff
@@ -74,12 +74,22 @@ contract PBHEntryPointImplV1 is IPBHEntryPoint, WorldIDImpl, ReentrancyGuard {
     /// @notice Thrown when the hash of the user operations is invalid
     error InvalidHashedOps();
 
+    /// @notice Thrown when the gas limit for a PBH multicall transaction is exceeded
+    error GasLimitExceeded();
```

Comment:
Since the max gas limit can be updated, lets emit the current gas limit with the error.